addons/mail/static/src/discuss/gif_picker/gif_picker.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="discuss.GifPicker" owl="1">
+        <div class="o-mail-GifPicker bg-white d-flex flex-column w-100 p-2" t-att-class="{ 'h-100': !store.isSmall }" t-attf-class="{{ props.className }}">
+            <div class="d-flex align-items-center">
+                <i t-if="!state.showCategories" aria-label="back" t-on-click="openCategories" class="fa fa-arrow-left cursor-pointer p-2 fs-5"/>
+                <input class="form-control border-0 flex-grow-1" placeholder="Search for a gif" t-on-input="onInput" t-model="state.searchTerm" t-ref="autofocus" />
+                <i class="oi oi-search p-2 fs-5" title="Search..." role="img" aria-label="Search..."></i>
+            </div>
+            <div class="overflow-y-auto overflow-x-hidden" t-ref="scroller">
+                <div t-if="state.loadingError" class="text-center">
+                    <span class="o-discuss-GifPicker-error d-block">😵‍💫</span>
+                    <span class="fs-5 text-muted d-block">Failed to load gifs...</span>
+                </div>
+                <div t-if="state.showCategories and !state.loadingError" class="row row-cols-2 gy-2 gx-2" aria-label="list">
+                    <div t-if="!store.guest" class="col">
+                        <div class="position-relative ratio ratio-16x9 cursor-pointer" t-on-click="onClickFavoritesCategory" aria-label="list-item">
+                            <img
+                                t-if="state.favorites.length > 0"
+                                class="o-discuss-GifPicker-category"
+                                t-att-src="state.favorites[0].media_formats.tinygif.url"
+                                loading="lazy"
+                            />
+                            <div class="position-absolute w-100 h-100 top-0 start-0 text-center o-text-white align-middle fs-2 o-bg-black bg-opacity-50">
+                                Favorites
+                            </div>
+                        </div>
+                    </div>
+                    <t t-foreach="state.categories" t-as="category" t-key="category.name">
+                        <div class="col">
+                            <div class="position-relative ratio ratio-16x9 cursor-pointer" t-on-click="() => this.onClickCategory(category)" aria-label="list-item">
+                                <img class="o-discuss-GifPickerCategory-img img-fluid" t-att-src="category.image" loading="lazy"/>
+                                <div class="position-absolute w-100 h-100 top-0 start-0 text-center o-text-white align-middle fs-2 o-bg-black bg-opacity-50">
+                                    <t t-out="category.name"/>
```

Comment:
I made background color black (regardless of theme).
![Screenshot 2023-06-05 at 13 58 46](https://github.com/odoo/odoo/assets/6569390/894748a4-d552-4ee4-8a9b-0192eec22c90)


`bg-black` turns white in dark mode, so it didn't look good:
![Screenshot 2023-06-05 at 13 59 52](https://github.com/odoo/odoo/assets/6569390/a44716e7-7fe3-42ee-af9b-0cc9881491f4)